build.sbt

```diff
@@ -199,7 +206,7 @@ lazy val client = (project in file("target/clients/java"))
     // OpenAPI generation specs
     openApiInputSpec := (file(".") / "api" / "all.yaml").toString,
     openApiGeneratorName := "java",
-    openApiOutputDir := (file("target") / "clients" / "java").toString,
```

Comment:
We move the openapi generated code from the project root directory to the `clients/java/target`, which make more sense because it's a Java SDK only.   it won't make other git changes, since all the generated code are excluded from git management.